src/main/java/org/broadinstitute/consent/http/resources/ElectionReviewResource.java

```diff
@@ -75,7 +75,7 @@ public ElectionReview getAccessElectionReviewByReferenceId(@PathParam("electionI
             dataSetId.addAll(dar.getData().getDatasetIds());
         }
         Consent consent = consentService.getConsentFromDatasetID(dataSetId.get(0));
-        ElectionReview accessElectionReview = service.describeElectionReviewByElectionId(electionId, isFinalAccess);
+        ElectionReview accessElectionReview = service.describeElectionReviewByElectionId(electionId, null);
```

Comment:
I think using null for an argument as a generic anti-pattern. We should strive to avoid it if at all possible. That said, it isn't always possible and sometimes we need to handle legacy code this way.